tap_github/schemas/pull_requests.json

```diff
@@ -8,6 +8,999 @@
+                  "type": ["null", "string"]
+                },
+                "statuses_url": {
+                  "type": ["null", "string"]
+                },
+                "subscribers_url": {
+                  "type": ["null", "string"]
+                },
+                "subscription_url": {
+                  "type": ["null", "string"]
+                },
+                "tags_url": {
+                  "type": ["null", "string"]
+                },
+                "teams_url": {
+                  "type": ["null", "string"]
+                },
+                "trees_url": {
+                  "type": ["null", "string"]
+                },
+                "clone_url": {
+                  "type": ["null", "string"]
+                },
+                "mirror_url": {
+                  "type": ["null", "string"]
+                },
+                "hooks_url": {
+                  "type": ["null", "string"]
+                },
+                "svn_url": {
+                  "type": ["null", "string"]
+                },
+                "homepage": {
+                  "type": ["null", "string"]
+                },
+                "language": {
+                  "type": ["null", "string"]
+                },
+                "forks_count": {
+                  "type": ["null", "integer"]
+                },
+                "stargazers_count": {
+                  "type": ["null", "integer"]
+                },
+                "watchers_count": {
+                  "type": ["null", "integer"]
+                },
+                "size": {
+                  "type": ["null", "integer"]
+                },
+                "default_branch": {
+                  "type": ["null", "string"]
+                },
+                "open_issues_count": {
+                  "type": ["null", "integer"]
+                },
+                "is_template": {
+                  "type": ["null", "boolean"]
+                },
+                "topics": {
+                  "type": ["null", "array"],
+                  "items": {
+                    "type": ["null", "string"]
+                  }
+                },
+                "has_issues": {
+                  "type": ["null", "boolean"]
+                },
+                "has_projects": {
+                  "type": ["null", "boolean"]
+                },
+                "has_wiki": {
+                  "type": ["null", "boolean"]
+                },
+                "has_pages": {
+                  "type": ["null", "boolean"]
+                },
+                "has_downloads": {
+                  "type": ["null", "boolean"]
+                },
+                "archived": {
+                  "type": ["null", "boolean"]
+                },
+                "disabled": {
+                  "type": ["null", "boolean"]
+                },
+                "visibility": {
+                  "type": ["null", "string"]
+                },
+                "pushed_at": {
```

Comment:
Here as well, add "format": "date-time" for the fields created_at,updated_at, and pushed_at as the format restrictions are mentioned in the API doc.
Please check other files as well for the same change.

Reply:
Actually in the API doc there are 4 fields named `pushed_at` two of which ar mentioned as pure string and other 2 are mentioned as date-time formats